README.md

```diff
@@ -10,42 +10,42 @@ It needs an apt based system like Ubuntu. Also the [stuvusIT.nginx](https://gith
+| `roundcube_skin`                    | :heavy_multiplication_x: | `larry`                                                                     | Theme to be used                                                                                                                                                                                |
+| `roundcube_log_driver`              | :heavy_multiplication_x: | `syslog`                                                                    | Where should roundcube log to.                                                                                                                                                                  |
+| `roundcube_working_dir`             | :heavy_multiplication_x: | `/opt/roundcube`                                                            | Working dir for this installation                                                                                                                                                               |
+| `roundcube_unarchive_dir`           | :heavy_multiplication_x: | `{{ roundcube_working_dir }}/roundcubemail-{{ roundcube_install_version }}` | Where should roundcube be extracted to                                                                                                                                                          |
+| `roundcube_managesieve_host`        | :heavy_multiplication_x: | `localhost`                                                                 | Host of the sieve server.                                                                                                                                                                       |
+| `roundcube_managesieve_port`        | :heavy_multiplication_x: | `4190`                                                                      | Port of the sieve server.                                                                                                                                                                       |
+| `roundcube_mail_domain`             | :heavy_multiplication_x: | ` `                                                                         | This domain will be used to form e-mail addresses of new users.                                                                                                                                 |
+| `roundcube_roundcube_extra_options` | :heavy_multiplication_x: | `[]`                                                                        | List of dicts each with a value and key option that will be written into the config file                                                                                                        |
 For more information please read the [roundcube default config file](https://github.com/roundcube/roundcubemail/blob/master/config/defaults.inc.php)
```

Comment:
This line somehow slipped into the markdown table, otherwise fine.

Reply:
fixed @SlothOfAnarchy pls rereview